### PR TITLE
implements ability to open create space/room modals

### PR DESCRIPTION
**Pull request by builat**
- relevant issue: https://github.com/netdata/product/issues/727

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "cloud-iframes",
   "version": "0.1.0",
   "private": true,
-  "homepage": "/sso",
+  "homepage": ".",
```
Comment (Azein) [inline package.json:5]: Should it be commited?

File: src/iframes/space-bar/spaces-bar.tsx [hunk 2]
```diff
@@ -1,58 +1,61 @@
-import React, { useState, useEffect } from "react"
-import { useMount } from "react-use"
-import { Button } from "@netdata/netdata-ui"
+import React, { useState, useEffect } from "react";
```
Comment (Azein) [inline src/iframes/space-bar/spaces-bar.tsx:1]: There is no prettierrc in the repo, right? There are a lot of changes applied by your default prettier settings, I think. Let's see if this is the case.

File: package.json (hunk 1 above)
Comment (builat) [reply package.json:5]: Yes, this is discussed with @jacekkolasa

File: src/iframes/space-bar/spaces-bar.tsx (hunk 2 above)
Comment (builat) [reply src/iframes/space-bar/spaces-bar.tsx]: done

**Pull request merged**